File: 240115/007/007.java
import java.util.Scanner;
class student{
    String name, alpha, age;

    public student(String name, String alpha, String age){
        this.name = name;
        this.alpha = alpha;
        this.age = age;
    }
}

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // 여기에 코드를 작성해주세요.
        String name = sc.next();
        String alpha = sc.next();
        String age = sc.next();
        student student1 = new student(name,alpha,age);
        System.out.println("secret code :"+" "+student1.name);
        System.out.println("meeting point :"+" "+student1.alpha);
        System.out.println("time :"+" "+student1.age);
    }
}

File: 240115/007/007.py
a, b, c = map(str,input().split())
c = int(c)

class student:
    def __init__ (self,a,b,c):
        self.a = a
        self.b = b
        self.c = c

tal = student(a,b,c)

print("secret code :",tal.a)
print("meeting point :",tal.b)
print("time :",tal.c)